gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-rc-1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-8.10-20240701001954+0000-bin.zip
```

Comment:
❓ depending on the snapshot wrapper bothers me. This means our builds are harder to reproduce. I don't feel pleasuring the formatted worth it. I'd delay this change

Reply:
Makes sense to me, I can leave this PR open until there is a new, non-snapshot distribution available.

Reply:
You can trigger a nightly promotion yourself in these cases if you don't wanna wait.

Reply:
That would still be a snapshot, wouldn't it? Well, "snapshot distribution"... But now that I give it more thought, these aren't really snapshots, are they? None of them will ever be changed or overwritten, so I'm not sure that the initial concern about the builds being harder to reproduce is valid. Wdyt?

Reply:
> None of them will ever be changed or overwritten,

They are deleted after some time, IRC